apps/admin-portal/src/components/identity-providers/settings/attribute-management/uri-attributes-settings.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Field, Forms } from "@wso2is/forms";
+import { Heading, Hint } from "@wso2is/react-components";
+import _ from "lodash";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Divider, Grid } from "semantic-ui-react";
+import { DropdownOptionsInterface } from "../attribute-settings";
+
+interface AdvanceAttributeSettingsPropsInterface {
+    dropDownOptions: DropdownOptionsInterface[];
+    initialSubjectUri: string;
+    initialRoleUri: string;
+    claimMappingOn: boolean;
+    updateRole: (roleUri: string) => void;
+    updateSubject: (subjectUri: string) => void;
+    triggerSubmit: boolean;
+}
+
+export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPropsInterface> = (
+    props
+): ReactElement => {
+
+    const {
+        dropDownOptions,
+        initialSubjectUri,
+        initialRoleUri,
+        claimMappingOn,
+        updateRole,
+        updateSubject,
+        triggerSubmit
+    } = props;
+
+    const getValidatedInitialValue = (initialValue: string) => {
+        return _.find(dropDownOptions, option => option?.value === initialValue) !== undefined ? initialValue : "";
+    }
+
+    return (
+        <>
+            <Grid.Row>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 10 }>
+                    <Divider/>
+                    <Divider hidden/>
+                </Grid.Column>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 8 }>
+                    {/* eslint-disable-next-line @typescript-eslint/no-empty-function */}
+                    <Forms submitState={ triggerSubmit } onSubmit={ (): void => { } }>
```

Comment:
Can't we use the Form as the outermost element?

Reply:
If they are updating for each change we can get rid of the Form or it is better to have proper onSubmit function where we update those values

Reply:
Fixed in 73f0cc02c52d71ca5acc51cf4120b806799a9d3d